input/intro-notes/StructureDefinition-au-diagnosticreport-intro.md

```diff
@@ -7,5 +7,5 @@
   - the individual component examinations are referenced by that grouping Observation in `Observation.hasMember` and not directly referenced in `DiagnosticReport.result`
   - `DiagnosticReport.code` and the study / panel Observation `Observation.code` should be the same concept if the report contains only the results of that study / panel
 - See each Identifier profile page for guidance related to that identifier type.
-- See the [AU Base Pathology Report](StructureDefinition-au-pathologyreport.html) profile for guidance on representing a pathology report issued by the diagnostic service provider.
-- See the [AU Base Diagnostic Imaging Report](StructureDefinition-au-imagingreport.html) profile for guidance on representing an imaging report issued by the diagnostic service provider.
\ No newline at end of file
+- See the preferred [AU Base Pathology Report](StructureDefinition-au-pathologyreport.html) profile for guidance on representing a pathology report issued by the diagnostic service provider.
+- See the preferred [AU Base Diagnostic Imaging Report](StructureDefinition-au-imagingreport.html) profile for guidance on representing an imaging report issued by the diagnostic service provider.
```

Comment:
In the ticket, preferred is italicised.

Reply:
Confirmed with @brettesler-ext that intention in resolution was to highlight that this is a new word and not intended to be italicised in IG.

Reply:
cool